JSON HW.playground/Contents.swift

```diff
@@ -1,3 +1,16 @@
-import UIKit
+import Foundation
 
-var greeting = "Hello, playground"
+let blackLotus = "Black Lotus"
+let opt = "Opt"
```

Comment:
Чтобы выводить то и то можно сделать следующим образом
```suggestion
let blackLotus = "Black Lotus|Opt"
```

Reply:
Но тут будет и подвох, из-за которого придется перешаманить метод декодирования